bokehjs/examples/anscombe/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "noImplicitAny": true,
+    "noEmitOnError": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
```

Comment:
Are we really using node module resolution for Bokeh? It looks like we just have a global "Bokeh" namespace, which is fine, but doesn't require this compiler flag.